src/components/auth/SignUp.js

```diff
@@ -47,6 +47,14 @@ const SignUp = (props) => {
             console.log(err);
         })
     };
+
+    // event handler to call pop up window for google sign up
+    const authWithGoogle = (e) => {
+        e.preventDefault();
+        window.open('http://localhost:4000/auth/google', '_self');
```

Comment:
Noice. This should probably be set to an environment variable so it's easier to deal with when we set up our domain / new google credentials.